### ISSUE TITLE
As the department of education, I want analytics in order to know how well the API is working to meet our goals

### ISSUE THREAD

**Issue by ErieMeyer** [18F/college-choice#205]
Track API analytics to know:

- Calls to the API
- Unique API users
[More?] cc @ultrasaurus @gbinal

**Comment by jackiekazil** [comment]
* stats by api key
* stats by user id (of api key)
* frequency of api use by app (& all apps)
* geo-origin of all the thing above 
* stats on url where the app is hosted
* gets or posts (if posts are offered)

**Comment by LisaGee** [comment]
@ErieMeyer Let's break these into separate stories and prioritize them. Thanks.

**Comment by barkimedes** [comment]
Need to split the other tasks into individual tasks; so the parts we originally wanted are done, but I'll move the remaining into backlog.

**Comment by abisker** [comment]
@barkimedes has this been split yet? Should we close this one out, or...?

**Comment by barkimedes** [comment]
@abisker no, I don't think I ever got to it. Rather than splitting them up now, I think the critical ones are done; these are lower priority. So I'm going to lower the priority of this story as a whole. And when someone actually gets to it, they can break it up into even more stories. Just to preserve my sanity by not creating 6 more stories right now :smile:
